htdocs/categories/categorie.php

```diff
@@ -419,7 +419,7 @@
 		// Ref
 		print '<tr><td width="20%">'.$langs->trans("Ref").'</td>';
 		print '<td class="valeur">';
-		print $form->showrefnav($member,'id');
+		print $form->showrefnav($member,'id','','1','rowid','ref','','&type=3');
```

Comment:
You're right. How to do this?
    Alain Rihs
      
    
    Le 22. 09. 15 09:20, Raphaël Doursenaud
      a écrit :
    
    
      In htdocs/categories/categorie.php:
      > @@ -419,7 +419,7 @@
>  		// Ref
>  		print '<tr><td width="20%">'.$langs->trans("Ref").'</td>';
>  		print '<td class="valeur">';
> -		print $form->showrefnav($member,'id');
> +		print $form->showrefnav($member,'id','','1','rowid','ref','','&type=3');

      You may want to use the local variable $type or
        the class constant Categorie::TYPE_MEMBER rather
        than a hardcoded 3 here. Both for readability and
        maintainability.
      —
        Reply to this email directly or view
          it on GitHub.